class.FlipPage.php

```diff
@@ -397,11 +401,11 @@ protected function addAllLinks()
             $this->addLink('Logout', $this->logoutUrl);
         }
         $aboutMenu = array(
```

Comment:
at this moment i just need to be able to change all links from burningflipside.com to localhost.. I don't want to do anything more invasive just yet..